test/integration/volume_snapshot_integration_test.go

```diff
@@ -0,0 +1,108 @@
+//go:build integration
+
+package integration
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/cloudscale-ch/cloudscale-go-sdk/v6"
+)
+
+func TestVolumeSnapshotIntegration_CRUD(t *testing.T) {
+	integrationTest(t)
+
+	ctx := context.Background()
+
+	// A source volume is needed to create a snapshot.
+	volumeCreateRequest := &cloudscale.VolumeRequest{
+		Name:   "test-volume-for-snapshot",
+		SizeGB: 50,
+		Type:   "ssd",
+	}
+	volume, err := client.Volumes.Create(ctx, volumeCreateRequest)
+	if err != nil {
+		t.Fatalf("Volume.Create: %v", err)
+	}
+
+	snapshotCreateRequest := &cloudscale.VolumeSnapshotRequest{
+		Name:         "test-snapshot",
+		SourceVolume: volume.UUID,
+	}
+	snapshot, err := client.VolumeSnapshots.Create(ctx, snapshotCreateRequest)
+	if err != nil {
+		t.Fatalf("VolumeSnapshots.Create: %v", err)
+	}
+
+	retrieved, err := client.VolumeSnapshots.Get(ctx, snapshot.UUID)
+	if err != nil {
+		t.Fatalf("VolumeSnapshots.Get: %v", err)
+	}
+	if retrieved.UUID != snapshot.UUID {
+		t.Errorf("Expected UUID %s, got %s", snapshot.UUID, retrieved.UUID)
+	}
+	if retrieved.Name != "test-snapshot" {
+		t.Errorf("Expected retrieved snapshot name 'test-snapshot', got '%s'", retrieved.Name)
+	}
+
+	snapshotUpdateRequest := &cloudscale.VolumeSnapshotUpdateRequest{
+		Name: "updated-snapshot",
+	}
+	err = client.VolumeSnapshots.Update(ctx, snapshot.UUID, snapshotUpdateRequest)
+	if err != nil {
+		t.Fatalf("VolumeSnapshots.Update: %v", err)
+	}
```

Comment:
for consistency with older tests:

we usually have a second test case with `_Update` that does an update

(which is misleading given the first test is usually `_CRUD` 🙈